plugins/modules/openssl_privatekey_info.py

```diff
@@ -130,6 +130,62 @@
         - Public key data. Depends on key type.
     returned: success
     type: dict
+    contains:
+        size:
+            description:
+                - Bit size of modulus (RSA) or prime number (DSA).
+            type: int
+            returned: When C(type=RSA) or C(type=DSA)
+        modulus:
+            description:
+                - The RSA key's modulus.
+            type: int
+            returned: When C(type=RSA)
+        exponent:
+            description:
+                - The RSA key's public exponent.
+            type: int
+            returned: When C(type=RSA)
+        p:
+            description:
+                - The C(p) value for DSA.
+                - This is the prime modulo which arithmetic takes place.
```

Comment:
```suggestion
                - This is the prime modulus upon which arithmetic takes place.
```
Modulo is an operation; Modulus is an operand.

Reply:
Heh. I should actually know that! Thanks for spotting :)